src/Voron/Impl/Transaction.cs

```diff
@@ -17,6 +17,8 @@
 using Voron.Data.PostingLists;
 using Constants = Voron.Global.Constants;
 using System.Diagnostics.CodeAnalysis;
+using static Sparrow.PortableExceptions;
+using static Voron.VoronExceptions;
```

Comment:
I am not a big fan of static imports either, but code looks much better without having to explicly call `PortableExceptions.Throw<whatever>` or `VoronExceptions.ThrowIfReadOnly(...)`

If I would be allowed to make a single exception to it's use, this would be the one.

Reply:
OK